kythe/typescript/indexer.ts

```diff
@@ -43,6 +43,10 @@ export interface IndexerHost {
    * not yet null-safe, so it hasn't been annotated with full types.)
    */
   getSymbolAtLocation(node: ts.Node): ts.Symbol|undefined;
+  /**
+   * getSymbolAtLocation is the same as ts.TypeChecker.getTypeAtLocation.
```

Comment:
Should we just expose the typeChecker instead then?

Reply:
Removed this method. We already have access to the typeChecker in indexer. If some plugin in future needs it - then we can consider adding it back.